src/main/java/com/gdsc/studyjamapi/service/UserService.java

```diff
@@ -67,4 +68,29 @@ public User getCurrentUser() {
   public UserResponse getCurrentUserResponse() {
     return UserMapper.INSTANCE.userToUserResponse(getCurrentUser());
   }
+
+  public UserResponse editUser(EditUserRequest request) {
+    // Check if the user exists by email
+    Optional<User> optionalUser = userRepository.findUserByEmail(request.getEmail());
+    if (!optionalUser.isPresent()) {
+      System.err.println("User not found with email: " + request.getEmail());
```

Comment:
We will use another logging method in the future. Currently, you don't need to print anything in the console. Please remove all `println` you have used.